shippable.yml

```diff
@@ -83,11 +83,11 @@ integrations:
       on_failure: always
       on_start: never
       on_pull_request: always
-    - integrationName: slack
```

Comment:
I'm very sorry, it's my fault. In order to make shippable test in my repository conveniently, I changed them, and I forgot to revert them when I pull new request. I will revert them in the next PR.